tests/cephadm/test_verify_cluster_health_after_reboot.py

```diff
@@ -46,6 +48,9 @@ def run(ceph_cluster, **kw):
                 raise ClusterNotHealthy(
                     f"Ceph cluster not healthy after rebooting {node.hostname}"
                 )
+        # Adding an explicit delay, because sudden reboots are causing the cluster Health to
+        # HEALTH WARN state.
+        sleep(30)
```

Comment:
Thanks for the comments. 

Instead of a blind sleep, can we check on the cluster status